server/hazard-service/src/main/java/edu/illinois/ncsa/incore/service/hazard/models/eq/utils/HazardUtil.java

```diff
@@ -109,20 +109,12 @@
     // TODO: This json with period is getting hard to read. Move to a static json file - should we use the demand type and unit
     //  definition constants defined here?.
     public final static JSONObject EARTHQUAKE_THRESHOLDS = new JSONObject("{ " +
-        "'" + PGA + "': {'0.0' : null}," +
-        "'" + PGV + "': {'0.0' : null}," +
-        "'" + PGD + "': {'0.0' : null}," +
-        "'" + SA + "':{'0.2': null," +
-        "'0.3': null," +
-        "'1.0': null," +
-        "'0.0': null," +
-        "}," +
-        "'" + SD + "':{'0.2': null," +
-        "'0.3': null," +
-        "'1.0': null," +
-        "'0.0': null," +
-        "}," +
-        "'" + SV + "': {'0.0' : null}," +
+        "'" + PGA + "': {'value': 0.01, 'unit': 'g'}," +
```

Comment:
@navarroc We haven't yet received the thresholds for all the EQ demands from scicomm. I have set them to 0.01 for now, but needs to be changed. do you think we can finalize these values before the release? Else, we can set them to null to disable threshold checking
https://opensource.ncsa.illinois.edu/confluence/display/INCORE1/Hazard+-+Demand+Types

Reply:
I'm checking with @jonglee1 to see if we can get some minimum values.

Reply:
Just to capture it here, while we are waiting for firm thresholds from the researchers, we will go with what you have.